menas/src/test/scala/za/co/absa/enceladus/menas/integration/TestInMemoryMenasAuthentication.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2018-2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.menas.integration
+
+import org.springframework.context.annotation.Primary
+import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder
+import org.springframework.security.crypto.password.NoOpPasswordEncoder
+import org.springframework.stereotype.Component
+import za.co.absa.enceladus.menas.auth.InMemoryMenasAuthentication
+
+@Component
+@Primary
+class TestInMemoryMenasAuthentication extends InMemoryMenasAuthentication {
+
+  override def configure(auth: AuthenticationManagerBuilder): Unit = {
+    super.configure(auth)
+
+    val inMemoryAuth = auth
+      .inMemoryAuthentication()
+      .passwordEncoder(NoOpPasswordEncoder.getInstance())
```

Comment:
Yeah, I know, but it doesn't seem to be going away. They've deprecated it so people don't use it in production, but since it is just for tests, I thought "no harm done". I'll replace it just to be rid of the deprecation warning.